app/controllers/schools/availability_preferences_controller.rb

```diff
@@ -4,7 +4,9 @@ def edit
   end
 
   def update
```

Comment:
Noticed on the select availability preference form if you submit it without selecting an option it blows up with missing parameter, might be worth adding a hidden field set to nil to the form, or using params.fetch

Reply:
👍🏽